src/artifact/v3/manifest/manifest.cpp

```diff
@@ -55,9 +55,11 @@ namespace {
 string MaybeStripSuffix(string s, vector<string> suffixes) {
 	auto s_ {s};
 	for (const auto &suffix : suffixes) {
-		if (s.substr(s.size() - suffix.size()) == suffix) {
-			s_.erase(s.size() - suffix.size());
-			return s_;
+		if (s.size() > suffix.size()) {
```

Comment:
I take it this means that if the suffix matches the string in `s` exactly, then it will not be removed? That's probably correct, but just checking.

Reply:
Yes. The reasoning is that if it matches exactly a suffix it is _not_ a suffix :wink: